components/partners/BecomeAPartnerForm.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div class="card">
+        <div class="text-dark p-2">
+            <div class="card-body">
+                <form>
+                    <div class="row mb-3">
```

Comment:
Please make it work like here: https://github.com/kestra-io/kestra.io/blob/main/components/contact/Header.vue#L68-L68